modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -41,107 +45,112 @@
   $backup_enabled = false
 ) {
 
-  validate_re($version, '^\d+\.\d+\.\d+$', 'govuk_elasticsearch::version must be in the form x.y.z')
-  include augeas
-  validate_bool($manage_repo)
+  if $enabled {
```

Comment:
It looks like the 'enable' parameter is basically invalidating the whole class, isn't it? This is the same as not including the class in first place. I think it's better to configure a flag in govuk_ci::agent directly to not include this class if we don't want to